DNN Platform/DotNetNuke.Abstractions/Logging/LoggingProviderReturnType.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+namespace DotNetNuke.Abstractions.Logging
+{
+    /// <summary>
+    /// Logging provider return type.
+    /// </summary>
+    public enum LoggingProviderReturnType
+    {
+        /// <summary>
+        /// Log info objects.
+        /// </summary>
+        LogInfoObjects = 0,
+
+        /// <summary>
+        /// XML.
+        /// </summary>
+        XML = 1,
```

Comment:
```suggestion
        Xml = 1,
```

Reply:
Fixed in latest code push, resolving thread